tests/test_fs_event.py

```diff
@@ -0,0 +1,106 @@
+import asyncio
+import os.path
+import tempfile
+
+from uvloop import _testbase as tb
+from uvloop.loop import FileSystemEvent
+
+
+class Test_UV_FS_EVENT_CHANGE(tb.UVTestCase):
+    async def _file_writer(self):
+        f = await self.q.get()
+        while True:
+            f.write('hello uvloop\n')
+            f.flush()
+            x = await self.q.get()
+            if x is None:
+                return
+
+    def fs_event_setup(self):
+        self.change_event_count = 0
+        self.fname = ''
+        self.q = asyncio.Queue()
+
+    def event_cb(self, ev_fname: bytes, evt: FileSystemEvent):
+        _d, fn = os.path.split(self.fname)
+        self.assertEqual(ev_fname, fn)
+        self.assertEqual(evt, FileSystemEvent.CHANGE)
+        self.change_event_count += 1
+        if self.change_event_count < 4:
+            self.q.put_nowait(0)
+        else:
+            self.q.put_nowait(None)
+
+    def test_fs_event_change(self):
+        self.fs_event_setup()
+
+        async def run(write_task):
+            self.q.put_nowait(tf)
+            try:
+                await asyncio.wait_for(write_task, 4)
+            except asyncio.TimeoutError:
+                write_task.cancel()
+            self.loop.stop()
+
+        with tempfile.NamedTemporaryFile('wt') as tf:
+            self.fname = tf.name.encode()
+            h = self.loop._monitor_fs(tf.name, self.event_cb)
+            self.assertFalse(h.cancelled())
+            try:
+                self.loop.run_until_complete(run(
+                    self.loop.create_task(self._file_writer())))
+                h.cancel()
+                self.assertTrue(h.cancelled())
+            finally:
+                self.loop.close()
```

Comment:
Don't we automatically close loops in `tearDown`?

Reply:
Right, fixed!